src/features/CourtCaseFilters/AppliedFilters.tsx

```diff
@@ -93,6 +106,19 @@ const AppliedFilters: React.FC<Props> = ({ filters }: Props) => {
               />
             </li>
           </If>
+          <If condition={!!filters.customDateFrom && !!filters.customDateTo}>
+            <li>
+              <FilterTag
+                tag={
+                  `${format(filters.customDateFrom || new Date(), "dd/MM/yyyy")} - ${format(
```

Comment:
I'm wondering if it's worth creating a global variable for `dd/MM/yyyy`. We've got `dateFormatString` in a different file.